test/faktory_worker/push_pipeline_test.exs

```diff
@@ -54,19 +56,10 @@ defmodule FaktoryWorker.PushPipelineTest do
     end
   end
 
-  defp default_child_spec() do
-    %{
-      id: FaktoryWorker.PushPipeline,
-      start: {FaktoryWorker.PushPipeline, :start_link, [[name: FaktoryWorker]]},
-      type: :supervisor
-    }
-  end
-
   defp get_child_spec_config(child_spec) do
     %{
       id: FaktoryWorker.PushPipeline,
-      start: {FaktoryWorker.PushPipeline, :start_link, [config]},
-      type: :supervisor
```

Comment:
Removed the `type: :supervisor` since we're not concerned with testing this since it's a Broadway implementation detail. All we really care about is that we have set the config and named the pipeline correctly.